stake-pool/cli/src/main.rs

```diff
@@ -719,6 +817,67 @@ fn main() {
                     .help("Stake account to receive SOL from the stake pool. Defaults to a new stake account."),
             )
         )
+        .subcommand(SubCommand::with_name("set-staking-auth").about("Changes staking authority of one of the accounts from the stake pool.")
+            .arg(
+                Arg::with_name("pool")
+                    .long("pool")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Stake pool address."),
+            )
+            .arg(
+                Arg::with_name("stake_account")
+                    .long("stake-account")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Stake account address to change staking authority."),
+            )
+            .arg(
+                Arg::with_name("new_staker")
+                    .long("new-staker")
+                    .validator(is_pubkey)
+                    .value_name("ADDRESS")
+                    .takes_value(true)
+                    .required(true)
+                    .help("Public key of the new staker account."),
```

Comment:
The CLI document mentions that this is optional and defaults to the owner, which makes sense to me.  Do we no longer want to do that?

Reply:
Since our discussion on how to deal with Staker authority (which prevents withdraw from happening if changed) did not finish I decided to keep it required, so the authority will not be changed accidentally.

Reply:
Ok that's fine, we can always change that later if we need to.